phoenicis-configuration/src/main/java/org/phoenicis/configuration/localisation/Localisation.java

```diff
@@ -74,4 +74,13 @@ public static String tr(String text, Object[] objects) {
         return getI18n().tr(text, objects);
     }
 
+    /**
+     * translate Translatable object
+     * @param translatable Translatable object
+     * @return translated object
+     */
+    public static Translatable tr(Translatable translatable) {
```

Comment:
I'm still missing in the javadoc, what "translating" means in this case.
I would suggest to add something like:

> Translates the given Translatable object, by creating a new instance of it with its corresponding translations